.github/workflows/pr-changelog-check.yml

```diff
@@ -12,4 +12,4 @@ jobs:
 
     - name: Changelog check
       if: ${{ !(contains(github.event.pull_request.labels.*.name, 'changelog-not-required') || contains(github.event.pull_request.labels.*.name, 'Design') || contains(github.event.pull_request.labels.*.name, 'Website') || contains(github.event.pull_request.labels.*.name, 'Documentation'))}}
-      run: ./hack/changelog-check.sh
+      run: ./hack/release-tools/changelog-check.sh
```

Comment:
`changelog-check.sh` is the script that verifies PRs have a changelog associated with them.
This is not part of the release tooling.